pkg/notifications/types.go

```diff
@@ -714,9 +793,6 @@ type AiNotificationsDestination struct {
 	UpdatedBy int `json:"updatedBy"`
 }
 
-// EntityGUID - An encoded Entity GUID
-type EntityGUID string
-
```

Comment:
We might not want to delete this type, as I find that this is being referenced in tests in the Terraform Provider as `notifications.EntityGUID` - please take a look and revert this if necessary.

Reply:
Yeah that's a mistake, I'll revert it

Reply:
Upon taking another look it seems like tutone generate simply moved it to line 1279, kinda weird but it's probably best keeping the what tutone did.